detecthttp/pytest.py

```diff
@@ -70,11 +70,16 @@ def pytest_runtest_call(item):
 
     if enabled:
         cassette_manager.__exit__(*sys.exc_info())
-        if len(cassette):
+        requested_host = []
```

Comment:
nit: should be plural right? maybe like `requested_hosts`?

Reply:
Right, thanks